src/main/java/ch/cern/eam/wshub/core/services/grids/entities/GridRequest.java

```diff
@@ -1,12 +1,15 @@
 package ch.cern.eam.wshub.core.services.grids.entities;
 
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
 import net.datastream.schemas.mp_functions.gridrequest.GRID;
 
 import javax.xml.bind.annotation.XmlElement;
 import javax.xml.bind.annotation.XmlElementWrapper;
 import java.io.Serializable;
 import java.util.*;
 
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
I believe we should remove it as it is not the case today and not the case for all remaining entities.

Reply:
Was used for ignoring the getRequestKey() method. Now a JsonIgnore annotation is applied to the method directly,